lib/ansible/module_utils/online.py

```diff
@@ -0,0 +1,116 @@
+import json
```

Comment:
Need to put moudle_utils files under a BSD license.  (See lib/ansible/module_utils/basic.py for an example of the header we use).

Also need boilerplate:
```

from __future__ import (absolute_import, division, print_function)
__metaclass__ = type
```